libs/perun/components/src/lib/resources-list/resources-list.component.ts

```diff
@@ -47,7 +47,7 @@ export class ResourcesListComponent implements AfterViewInit, OnChanges {
 
   private sort: MatSort;
 
-  displayedColumns: string[] = ['select', 'id', 'name', 'facility', 'tags', 'description'];
+  displayedColumns: string[] = ['select', 'id', 'name', 'vo', 'facility', 'tags', 'description'];
```

Comment:
In previous comment I also asked you to remove Input "hideColumns" and instead of this use Input "displayedColumns" where every component that use resource list would define which columns would like to use.

Reply:
Also for the best code readability move the rows 
`@Input()
  displayedColumns: string[] = ['select', 'id', 'name', 'vo', 'facility', 'tags', 'description'];`
right under another Inputs